src/browser/provider/module-loader.js

```diff
@@ -0,0 +1,18 @@
+const BROWSER_PROVIDER_SCOPED_PACKAGE_RE = /^@\S+\/testcafe-browser-provider\S/;
+
+export default class BrowserProviderModuleLoader {
+    loadModule (providerName) {
+        var providerObject = null;
+
+        if (this.isPrivateModule(providerName))
+            providerObject = require(`${providerName}`);
```

Comment:
I think it's better to extract `scope` from RegExp and use
```
providerObject = require(`${scope}/testcafe-browser-provider-${providerName}`);
```
here

Reply:
Updated - please have a look @AndreyBelym and let me know any feedback.  Note that I've passed all the test-server tests but on my machine (windows 10 XPS 15, and on my linux VM) I can't run test-functional-local and test-client... the functional tests break in weird ways that aren't replicable in the build process, and the test-client never start after booting the qunit server on port 2000?